packages/cspell-lib/src/index.test.ts

```diff
@@ -13,8 +13,4 @@ describe('Validate the cspell API', () => {
             expect(results.map((to) => to.text)).toEqual(expect.arrayContaining(['Jansons']));
         });
     });
-
-    test('clearCachedSettings', () => {
```

Comment:
This is actually a test.
It might be better as:
```
    test('clearCachedSettings', () => {
        return expect(cspell.clearCachedFiles()).resolves.not.toThrow();
    });
```

Reply:
OK, too your code, but flipped it to async/await